src/components/Scorecards/ScorecardDetailsPage/ScorecardRulesCard/ScorecardRuleRow.tsx

```diff
@@ -67,8 +73,37 @@ export const ScorecardRuleRow = ({ rule }: ScorecardRuleRowProps) => {
           </>
         </Collapse>
       </Grid>
-      <Grid item lg={2}>
-        {hasWeight && <b>{rule.weight}</b>}
+      <Grid item xs={4}>
+        {hasWeight && (
+          <Typography className={classes.rule}>{rule.weight}</Typography>
+        )}
+      </Grid>
+      <Grid item xs={4}>
```

Comment:
Not a huge deal but I wonder if we should but this in the collapsible with the desc. It looks busy as is and could look busier if there's lots of groups / a long query.

Reply:
Good point, do you mean only the filter in a collapsible and leave desc/weight the same?

Reply:
Desc is already in collapsible right? So put it below that in the same collapsible

Reply:
This is way better! See new screenshots in PR description.